css/styles.css

```diff
@@ -158,3 +158,598 @@ html {
 .social-icons:hover {
   color: #8a2be2;
 }
+
+/* Work-section css start from here */
+
+.work-section {
+  width: 412px;
+  height: 850px;
+
+  /* border-bottom: 2px solid purple; */
+  display: grid;
+  grid-template-rows: 1fr 2fr 3fr;
+}
+
+.box-1 {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  font-family: "Crete Round", serif;
+  font-style: normal;
+  font-weight: 0;
+  font-size: 20px;
+  color: #1c1a19;
+  border-bottom: 1px solid #212121;
```

Comment:
There are a couple of colors in style.css file, which are being repeated.
To write DRY code, try not to repeat the same colors over and over again. To deal with repetition, you can define your color variables once, and then use them again. For Example:
```
  --color-primary: #272a31;
  --color-secondary: #ec5242;
  --color-tertiary: #d3d3d3;
  --color-white: #fff;
  --color-grey: #585858;
  --color-light-grey: #ebf0ee;
or like
  --primary: #272a31;
  --secondary: #ec5242;
  --tertiary: #d3d3d3;
```